react/components/BuyButton/index.js

```diff
@@ -128,10 +127,12 @@ BuyButton.propTypes = {
         PropTypes.string,
         PropTypes.number,
       ]).isRequired,
-      name: PropTypes.string.isRequired,
-      price: PropTypes.number.isRequired,
-      variant: PropTypes.string,
-      brand: PropTypes.string.isRequired,
+      options: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
Why you are removing the name, price, variant, brand...?

Reply:
porque não são usadas de maneira nenuma e no Rebuy não tem pq passar.

Como não são usadas no componente, retirei tudo para evitar perda de mais tempo e confusão (que eu tive)

Reply:
Mas são usadas pelo Pixel. :D 

So pra justificar o/